src/components/UI/docs/DocumentNav.tsx

```diff
@@ -24,7 +24,7 @@ export const DocumentNav: FC<Props> = ({ content }) => {
       <Text as='h5' textStyle='document-nav-title'>
         on this page
       </Text>
-      <Divider borderColor='primary' my={`4 !important`} />
+      <Divider borderColor='primary' my={`4 !important`} w="70%" />
```

Comment:
I'll go with `w="100%" (that already prevents the overflow), otherwise it'll be shorter than the content in some cases, e.g.:

<img width="250" alt="Screen Shot 2022-12-02 at 13 44 26" src="https://user-images.githubusercontent.com/948922/205343386-9b33a8d4-66e4-4354-a749-f6785293694a.png">

compare it with the last item `Building without a Go workflow`

Reply:
Okay... I'll take a closer look, since this divider was flowing off the right edge of the page on some viewports, which means the text may also be doing the same. May need to adjust the whole container.

Reply:
Okay, so I removed that 70% change, so the divider is full width.

The extra 2rem introduced by the gap between the content and this sidebar means this whole container, which was hard-coded to be `{48}` (192px) width, ended up shifting slightly off the right side at some view ports. Removing 2rem from this width fixes the issue, but then leaves this div unnecessarily narrow on larger screens.

I used a `clamp` function instead, which _targets_ a width here of 12.5%, clamped to a minimum of {40} (160px, 2rem smaller than before), and a max of {56} (224px) on larger screens.

Reply:
Checked with @nloureiro and he's good with this approach. PR should be good to go